webpack.mix.js

```diff
@@ -15,11 +15,20 @@ require('./node_modules/eclipsefdn-solstice-assets/webpack-solstice-assets.mix.j
 let mix = require('laravel-mix');
 mix.EclipseFdnSolsticeAssets();
 
+// required to resolve missing polyfills
```

Comment:
If this is going to be needed for all the sites, let's fix it in solstice-assets so that all sites can benefit: 
https://gitlab.eclipse.org/eclipsefdn/it/webdev/solstice-assets/-/blob/main/webpack-solstice-assets.mix.js